core/commands/resolve.go

```diff
@@ -62,27 +70,19 @@ Resolve te value of another name:
 			}
 		}
 
-		var name string
-
-		if len(req.Arguments()) == 0 {
-			if n.Identity == "" {
-				res.SetError(errors.New("Identity not loaded!"), cmds.ErrNormal)
-				return
-			}
-			name = n.Identity.Pretty()
```

Comment:
no longer default to the local identity? this is probably ok for `resolve`, but may feel asymmetric for users of `publish`